stdlib/public/runtime/MetadataLookup.cpp

```diff
@@ -34,7 +34,11 @@
 #include <link.h>
 #endif
 
+#if defined(_MSC_VER)
+#include <windows.h>
+#else
 #include <dlfcn.h>
```

Comment:
It seems like this header is only used on Apple systems.  Could you check?  If so, could you move it into the `#if defined(__APPLE__) && defined(__MACH__)` block above?